pkg/operator/bootstrap.go

```diff
@@ -205,7 +205,7 @@ func RenderBootstrap(
 		if err := os.MkdirAll(dirname, 0655); err != nil {
 			return err
 		}
-		if err := ioutil.WriteFile(path, b, 0655); err != nil {
```

Comment:
Need clarification on whether there was a reason for this file mode. It seems weird to set the executable bit for only group and other. I guess we'll see what ci says.

I had to change this because keepalived won't run with an executable config file, and I needed to maintain permissions when baremetal-runtimecfg templates files since we're using external scripts for keepalived healthchecks now.

Reply:
Sounds like this is fine, but I had to move it to its own PR because it creates a circular dependency with the baremetal-runtimecfg PR.

https://github.com/openshift/machine-config-operator/pull/1934